config/nrfconnect/chip-module/Kconfig.defaults

```diff
@@ -337,6 +337,9 @@ config MBEDTLS_SHA256_C
 config MBEDTLS_PK_C
     default y
 
+config MBEDTLS_PKCS5_C
```

Comment:
I don't see any `MBEDTLS_PKCS5_C` usage in the code. Why this is needed?

Reply:
Oh i found it!
https://github.com/nrfconnect/sdk-connectedhomeip/blob/master/src/crypto/CHIPCryptoPALmbedTLS.cpp#L358